pvlib/tests/test_irradiance.py

```diff
@@ -1406,3 +1407,10 @@ def test_louche():
     out = irradiance.louche(ghi, zenith, index)
 
     assert_frame_equal(out, expected)
+
+
+def test_SURFACE_ALBEDOS_deprecated():
+    with pytest.warns(pvlibDeprecationWarning, match='SURFACE_ALBEDOS has been'
+                      ' moved to the albedo module as of v0.11.0. Please use'
+                      ' pvlib.albedo.SURFACE_ALBEDOS.'):
+        irradiance.SURFACE_ALBEDOS
```

Comment:
@AdamRJensen suggests adding another test to ensure that `pvlib.irradiance.SURFACE_ALBEDOS` is identical to `pvlib.albedo.SURFACE_ALBEDOS`